### ISSUE TITLE
Firebug not working with firefox 56,57

### ISSUE THREAD

**Issue by hmbashar** [firebug/firebug#8093]
Firebug 2.0.9 is not showing the HTML in the inspector. Can show CSS styles of the element inspected but can't show the HTML in Firefox 56.0, and i can't install firebug with firefox 57 version, firebug is not support in firefox 57 version, please update firebug for firefox 57 version, i want to use always firebug because it's great helpful for my work.

Best Regards
Bashar

**Comment by thehung** [comment]
The Firebug extension isn't being developed or maintained any longer. Mozilla invite you to use the Firefox DevTools instead, which ship with Firebug.next: [https://developer.mozilla.org/en-US/docs/Tools](https://developer.mozilla.org/en-US/docs/Tools)

**Comment by uzielweb** [comment]
Firebug is better then DevTools for me. Please continue the support.

**Comment by hmbashar** [comment]
Firebug is better then DevTools for me.

**Comment by Storm-Engineer** [comment]
Why would they stop maintaining one of the top 5 most used plugin of all time???

Firebug is better than anything else. Please pass development to someone willing to continue.

**Comment by k4mrul** [comment]
Firebug is better than dev tool. Downgrading my firefox now

**Comment by ut4utc** [comment]
> Why would they stop maintaining one of the top 5 most used plugin of all time???
it's simple - Google bought the plug-in developer, so that the developer created the same extension in Chrome. Nothing personal, only money.

Firebug is better than dev tool and for me. 
I'm downgrading my firefox now.

**Comment by Colinsp** [comment]
Please continue development the inbuilt Firefox tool is rubbish in comparison.

**Comment by uniquesolution** [comment]
Why why why? Its one of the most popuplar developer tool for such a long time, why you need to stop its updates ?

**Comment by Dentoo** [comment]
So now we have to use the free inbuilt tool on this free browser? Wow what a terrible world lol

I also find firebug much better then the "inbuilt" tool, and I hope Mozilla are working on it to make it par with Firebug. 

Damned Google, if its true that you bought the dev, assholes lol, you limit good dev tools to a noob-browser (Chrome). If you want devs to use Chrome I suggest you make it more like Firefox ;)

**Comment by greenreptile** [comment]
Firefox is not usable anymore!

**Comment by GuenniWeb** [comment]
horrible - i liked the firebug most - and everything worked well. I do not downgrade but change now the browser.

**Comment by legitalk** [comment]
So FF 55 is my last, coz in 56 Firebug is no more... Good bye!

**Comment by ghost** [comment]
hi,so easy!Firebug upgrade。

**Comment by cscd98** [comment]
I've been using firebug for many years, it's a sad day to see it go. The developer tools in Firefox is no comparison.

**Comment by GolubovicM** [comment]
Without Firebug I will move to Chrome...:(

**Comment by Dentoo** [comment]
Sad when a great popular feature that a popular nonprofit organization uses and loves gets bought by a company that already is crap-ass wealthy and that made their money on advertising. Specially considering only reason Google want Chrome to be popular is so that they can monitor peoples behavior and serve them advertising in a more efficient way for them. I dont blame the Firebug dev for taking Googles money, hopefully he will take it and run, but I do blame Google for not taking responsibility for what they really are.

**Comment by legitalk** [comment]
@wangshoubai not true, firebug already is up-to-date... and try it, you can't see nothing in the DOM!

**Comment by Dentoo** [comment]
yeey awesome :) sorry for all complaining, thanks for bringing it back!

**Comment by uniquesolution** [comment]
Hay,

The firebug is still not working properly for the latest version 56.0.2 of firefox, its showing the same dead blank screen for HTML tab. :(

**Comment by ut4utc** [comment]
Yes, because today site has changed https://getfirebug.com/ and what the fuck? Developer say:

> Today, the work pioneered by the Firebug community through the last 12 years lives on in Firefox Developer Tools. Switch to the latest version of Firefox and see the very latest devtools updates.

Go to devtools? But **devtools is Big Shit**...

Developers do not want to change the extension, but changed the site :-) **shit**

**Comment by uniquesolution** [comment]
Hummm, 

Right so its time to move to firefox developer edition https://www.mozilla.org/en-US/firefox/developer/?utm_source=firebug&utm_medium=lp&utm_campaign=switch&utm_content=landingpage , really much improved dev panel. So its time to say bye to firebug and use this separate firefox version for developing purpose 👍

**Comment by NibirB** [comment]
Please continue the development of firebug. Always enjoyed working with it. Thanks for providing such an awesome tool.

**Comment by SahilSelenium** [comment]
Hi guys firebug is not working with the latest version. We all are facing this issue . we can do one one thing we need to install new version 57 of Mozilla installer.exe and you can download the old version of Firefox which supports firebug . so for selenium use you guys can use firebug on old version and you can launch the new version for running the script.

please check the screenshot.[you can see there are two different versions of firefox]
Here is the link for screenshot

https://i.stack.imgur.com/8ZnFr.png

**Comment by uniquesolution** [comment]
Hi guys the only solution at the moment is using two version of firefox altogether, I was using many very useful addons with firefox like firebug, Reminderfox, colorzilla etc. Was also using fireshot which is available for new version too but having limited features. So here is my solution:- 
Install latest firefox quantum in your pc then download portable firefox older version from here https://sourceforge.net/projects/portableapps/files/Mozilla%20Firefox%2C%20Portable%20Ed./Mozilla%20Firefox%2C%20Portable%20Edition%2048.0.2/ and just extract it any where in your pc. Create its shortcut on desktop, then right click it and go to Properties. In the window that open go to "Shortcut" tab and in target field write "D:\portapps\FirefoxPortable-48.0.2\FirefoxPortable.exe -P "My Profile" -no-remote, so it will be "your file path -P "My Profile" -no-remote" . This will allow you to run both old and new version simultaneously, so that you can use old addons in old version. REMEMBER always open older version first to whom you have added the "-P "My Profile" -no-remote", then open the new version.

**Comment by ut4utc** [comment]
I'm using now firefox v54 on Mac + firebug + some plugins for web develop and never update my ff browser in the future. Find v54 here [https://ftp.mozilla.org/pub/firefox/releases/](https://ftp.mozilla.org/pub/firefox/releases/)
But first what you need do after install - that is disable your Wi-Fi or Lan and then open **Firefox -> Preferences -> Advanced -> Update ->** and set option: **Never check for updates (not recommended: security risk**)
Then you can run Wi-Fi or Lan, because if you don't do this - firefox download latest version on background and after restart Fox you can see latest update :-)

**Comment by Dentoo** [comment]
@uniquesolution 
Thank you, it works :) 

Copied all files in Quantum /appdata/roaming/mozilla/firefox/profile/profilehere to portable data/profile and now FireFTP with all accounts is back! As well as Firebug.

Edit: For some reason the portable version updated itself even though I had turned updates OFF! It might have been because synch was still active (didnt think of that). Have re-reinstalled portable version and works again.

Edit:  Works no more since Mozilla are deadly decided to NOT have Firebug or ANYTHING else that people like anymore.

**Comment by Vishalkr1091** [comment]
Please provide the support of firebug and firepath in latest version of firefox. If someone will find the solution how we can use firebug with firefox quantum please let me know.

**Comment by Vencatraz** [comment]
Firefox team should continue to provide the support for the Add-ons 'Firebug' and 'Firepath' which is absolutely necessary and helpful for the automation QA's to identify the Xpaths of the Web Elements. Identifying the Xpaths of the Elements using the Firepath is user friendly and much easier when compared to the other Development tools.

**Comment by Dentoo** [comment]
You Didnt get the message? Mozilla dont give a shit about us.
